CUSTOM.md

```diff
@@ -393,3 +393,18 @@ This section is meant for users who are familiar with both the tts mod and progr
       - `isGain`: **bool** - true if the operation is a gain, false if it's a pay
       - `isUndo`: **bool** - true if the operation is being undone (right click), false otherwise
       - `amount`: **number** - amount of energy gained/paid (before bargain deductions); positive for gain or pay
+- Trigger to modify how a spirit's elements are counted
+  - Create object and tag with "Modify Elements"
+  - `modifyElements(params)`
+    - `params`: **table** - contains data about the player and their elements
+      - `color`: **string** - is the color of the player whose elements are being counted
+      - `elements`: **table** - a table of eight numbers, representing the number of each element in order
```

Comment:
should the documentation include a note about the format of the table, about how it's indexed by number not name, and include the order of the elements?

Reply:
I'll definitely mention that its indexed by number. I considered listing the order of the elements, but it seemed unnecessary to give a full list given the order appears on every power card in the game. But I guess there's no harm in listing it.